tests/testthat/test-csv-read.R

```diff
@@ -181,3 +181,18 @@ test_that("bad paths", {
     "failed to locate file"
   )
 })
+
+test_that("cache url tempfile", {
+  url <- "https://vincentarelbundock.github.io/Rdatasets/csv/AER/BenderlyZwick.csv"
```

Comment:
Please check the website status with `testthat::skip_if_offline()` <https://testthat.r-lib.org/reference/skip.html>